config/configmap/inferenceservice.yaml

```diff
@@ -69,6 +95,11 @@ data:
             "defaultImageVersion": "v0.3.0",
             "allowedImageVersions": [
                "v0.3.0"
+            ],
+            "supportedModelType": [
```

Comment:
@cliveseldon I am trying to extend the inferenceservice configMap to allow the explainer server to advertise the model types it can support. Does it look right to you?

Reply:
So when people create an InferenceService they can optionally add an explainer, then, they can add explainer TrainedModels afterwards? The explainer types for Alibi are AnchorImages, AnchorText, so are <algorithmn>-<input-type>. At present the `gcr.io/kfserving/alibi-explainer` image handles just 1 explainer type and model. Maybe I need to understand how we expect the first version to work with a loaded explainer model and work backwards.

Reply:
That's how I think it should work. If the parent InferenceService has an explainer, then the TrainedModel CR should also container an explainer section. Otherwise this TrainedModel CR should be rejected. 

Which explainer type gcr.io/kfserving/alibi-explainer handles at present?

Reply:
Alibi handle all 3 text, image, tabular. But the type of the algorithm will need to match,e .g. AnchorText for Text. Also there is the issue of whether the model is classifier, regression see https://docs.seldon.io/projects/alibi/en/latest/overview/algorithms.html#model-explanations for the matrix map of this to available algorithm.